tet/security/authentication.py

```diff
@@ -1161,8 +1156,10 @@ def revoke_other_tokens(self):
         if user is None or not self.auth_service.verify_password(
             user=user, password=payload.get("password", "")
         ):
+            self.registry.notify(RevokeOtherRefreshTokensFailedEvent(request=self.request))
```

Comment:
that does not actually signal the intent properly. You've got several things here: either user is None or then the password fails, and neither of them actually means the "operation failed"